job-dsls/jobs/osbs/bamoe_osbs_seed_cloud_jobs.groovy

```diff
@@ -1,6 +1,6 @@
 import org.kie.jenkins.jobdsl.Constants
 
-def folderPath = 'OSBS'
+def folderPath = 'OSBS/rhpam-images'
```

Comment:
@rgdoliveira do you know why did it fail?
the 'folder' doc does not have much information, and seems to support `dir/dir`

Reply:
It failed with the same error, I'm not sure whats going on. @lampajr any idea of what can be wrong?

@spolti Just a minor comment, I suppose in the `bamoe_osbs_seed_cloud_jobs.groovy` the folder name should be `bamoe-images` instead of `rhpam-images`?

Reply:
yes :)